nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/util/ClientResponseUtils.java

```diff
@@ -41,7 +40,7 @@ public static void drainClientResponse(final Response response) {
                 bis = new BufferedInputStream(response.readEntity(InputStream.class));
                 IOUtils.copy(bis, new NullOutputStream());
             } catch (final IOException ioe) {
-                logger.info("Failed clearing out non-client response buffer due to: " + ioe, ioe);
+                logger.info("Failed clearing out non-client response buffer", ioe);
```

Comment:
We should avoid logging an exception at info level.  In this case the output should be warn as it happening would be interesting to know why we weren't able to drain the response.